README.md

```diff
@@ -85,8 +86,13 @@ So you can use normal container parameter syntax.
 - service\_container is tested via simpletest for integration with Drupal. (run ./tests/run-simpletests.sh)
 - service\_container is tested via PHPUnit for code coverage. (run ./tests/run-coverage.sh)
 
-### Status
+### List of services that we've altered
 
-[![Build Status](https://travis-ci.org/LionsAd/service_container.svg?branch=7.x-1.x)](https://travis-ci.org/LionsAd/service_container)
-[![Coverage Status](https://coveralls.io/repos/LionsAd/service_container/badge.png?branch=7.x-1.x)](https://coveralls.io/r/LionsAd/service_container?branch=7.x-1.x)
-[![Gitter](https://badges.gitter.im/Join Chat.svg)](https://gitter.im/LionsAd/service_container?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+The following services contains files that has been altered from their Drupal 8 counterpart to make them compatible with Drupal 7.
+
+* Database
+* KeyValueStore
+* StringTranslation
```

Comment:
Can we add the direct filenames? That is important for security, reviews, etc.

e.g.

The list of files is like follows:

 (after the list of services)